iOS/Flows/Conversation/Manager/ConversationCollectionViewManager.swift

```diff
@@ -212,20 +197,20 @@ class ConversationCollectionViewManager: NSObject, UITextViewDelegate, Conversat
     private func getTopHeader(for section: ConversationSectionable,
                               at indexPath: IndexPath,
                               in collectionView: ConversationCollectionView) -> UICollectionReusableView? {
-
-        guard let index = section.firstMessageIndex, index > 0 else { return nil }
-
-        let moreHeader = collectionView.dequeueReusableHeaderView(LoadMoreSectionHeader.self, for: indexPath)
-
-        moreHeader.button.didSelect { [weak self] in
-            guard let `self` = self else { return }
-            Task {
-                await moreHeader.button.handleEvent(status: .loading)
-            }
-            self.didSelectLoadMore(for: index)
-        }
-
-        return moreHeader
+        return nil
```

Comment:
Not sure we need this header at all for threads. My preference would be to pull out all unnecessary elements and rebuild if needed. Thoughts?